cdk/lib/support-reminders.ts

```diff
@@ -1,15 +1,247 @@
+			functionName: `support-reminders-create-reminder-signup-${this.stage}-CDK`,
+			...sharedLambdaProps,
+		});
+
+		const reactivateRecurringReminderLambda = new GuLambdaFunction(this, "reactivate-recurring-reminder", {
+			handler: "reactivate-recurring-reminder/lambda/lambda.handler",
+			functionName: `support-reminders-reactivate-recurring-reminder-${this.stage}-CDK`,
+			...sharedLambdaProps,
+		});
+
+		const cancelRemindersLambda = new GuLambdaFunction(this, "cancel-reminders", {
+			handler: "cancel-reminders/lambda/lambda.handler",
+			functionName: `support-reminders-cancel-reminders-${this.stage}-CDK`,
+			...sharedLambdaProps,
+		});
+
+
+		// ---- API gateway ---- //
+		const supportRemindersApi = new GuApiGatewayWithLambdaByPath(this, {
+			app,
+			defaultCorsPreflightOptions: {
+				allowOrigins: Cors.ALL_ORIGINS,
+				allowMethods: Cors.ALL_METHODS,
+				allowHeaders: ["Content-Type"],
+			},
+			monitoringConfiguration: {
+				snsTopicName: "conversion-dev",
+				http5xxAlarm: {
+					tolerated5xxPercentage: 1,
+				}
+			},
+			targets: [
+				{
+					path: "/reactivate",
+					httpMethod: "POST",
+					lambda: reactivateRecurringReminderLambda,
+				},
+				{
+					path: "/create/recurring",
+					httpMethod: "POST",
+					lambda: createRemindersSignupLambda,
+				},
+				{
+					path: "/search",
+					httpMethod: "POST",
+					lambda: searchRemindersLambda,
+				},
+				{
+					path: "/create/one-off",
+					httpMethod: "POST",
+					lambda: createRemindersSignupLambda,
+				},
+				{
+					path: "/cancel",
+					httpMethod: "POST",
+					lambda: cancelRemindersLambda,
+				},
+			],
+		})
+
+
+		// ---- Scheduled lambda functions ---- //
+		new GuScheduledLambda(this, "signup-exports", {
+			handler: "signup-exports/lambda/lambda.handler",
+			functionName: `support-reminders-signup-exports-${this.stage}-CDK`,
+			rules: [
+				{
+					schedule: Schedule.cron({ hour: "00", minute: "05" }),
+				},
+			],
+			monitoringConfiguration: {
+				snsTopicName: "conversion-dev",
+				toleratedErrorPercentage: 1,
+			},
+			...sharedLambdaProps,
+		});
+
+		new GuScheduledLambda(this, "next-reminders", {
+			handler: "next-reminders/lambda/lambda.handler",
+			functionName: `support-reminders-next-reminders-${this.stage}-CDK`,
+			rules: [
+				{
+					schedule: Schedule.cron({ hour: "00", minute: "05" }),
+				},
+			],
+			monitoringConfiguration: {
+				snsTopicName: "conversion-dev",
+				toleratedErrorPercentage: 1,
+			},
+			...sharedLambdaProps,
+		});
+
+
+		// ---- DNS ---- //
+		// const certificateArn = `arn:aws:acm:eu-west-1:${this.account}:certificate/${props.certificateId}`;
+		//
+		// const cfnDomainName = new CfnDomainName(this, "DomainName", {
+		// 	domainName: props.domainName,
+		// 	regionalCertificateArn: certificateArn,
+		// 	endpointConfiguration: {
+		// 		types: ["REGIONAL"]
+		// 	}
+		// });
+		//
+		// new CfnBasePathMapping(this, "ApiMapping", {
```

Comment:
When testing I think we noticed that this id should be:

```suggestion
		// new CfnBasePathMapping(this, "BasePathMapping", {
```

Obviously this doesn't really matter when this code is commented out but I just thought I'd note that down while we still remember, as we'll need it for the next PR!

Reply:
Good catch! I've updated the ID in this PR 👍🏼